src/config/cloudinary.ts

```diff
@@ -0,0 +1,10 @@
+import { v2 as cloudinary } from 'cloudinary';
+
+cloudinary.config({
```

Comment:
I don't think it would be intentional but don't push your keys on github.
Always create a .env.example file to just push the variable name and use .env